Gruntfile.js

```diff
@@ -15,9 +15,17 @@ module.exports = function (grunt) {
                 dest: 'src/javascripts/test/fixtures/',
                 options: {
                     process: function(content) {
-                        return content.replace(/http\:\/\/localhost\:8080\//g, '/');
+                        return content.replace(/http\:\/\/localhost\:8000\//g, '/');
                     }
                 }
+            },
+            test_fakerest: {
+                src: 'node_modules/fakerest/dist/FakeRest.min.js',
```

Comment:
Because copy from example build directory to test build directory does not copy symlinks (which would be broken anyway).